istioctl/cmd/auth.go

```diff
@@ -100,12 +100,12 @@ Kubernetes cluster or from a yaml file specified from command line.
 THIS COMMAND IS STILL UNDER ACTIVE DEVELOPMENT AND NOT READY FOR PRODUCTION USE.
 `,
 		Example: `  # Upgrade the Istio authorization policy with service definition from the current k8s cluster:
-  istioctl experimental auth upgrade -f istio-authz-v1-policy.yaml
+  istioctl experimental auth upgrade -f istio-authz-v1-policy-1.yaml istio-authz-v1-policy-2.yaml
```

Comment:
Should it be `,` separated string? Not sure if whitespace works here.

Reply:
You're right. It won't work. Edited.